data_hub_api/docmaps/v2/codecs/docmaps.py

```diff
@@ -284,11 +308,13 @@ def iter_docmap_steps_for_query_result_item(query_result_item: ApiInput) -> Iter
                     query_result_item=query_result_item,
                     manuscript_version=manuscript_version
                 )
-        if manuscript_version['vor_publication_date'] and index == len(manuscript_versions) - 1:
-            yield get_docmaps_step_for_vor_published_status(
-                query_result_item=query_result_item,
-                manuscript_version=manuscript_version
-            )
+        if vor_versions and index == len(manuscript_versions) - 1:
```

Comment:
very minor: I'd maybe find it easier to follow if it was outside the loop. The if statement could then be something like (correct me if I am wrong):

```python
if manuscript_versions and vor_versions:
```